tests/drivers/build_all/can/testcase.yaml

```diff
@@ -16,6 +16,15 @@ tests:
       - arduino_gpio
     extra_args: SHIELD=tcan4550evm
     platform_allow: frdm_k64f
+  drivers.can.build_all.tcan4x5x.mikrobus:
+    extra_args: SHIELD=mikroe_can_fd_6_click
+    platform_allow: mikroe_stm32_m4_clicker
+  drivers.can.build_all.tcan4x5x.mikrobus.arduino:
+    depends_on:
+      - arduino_spi
+      - arduino_gpio
+    extra_args: SHIELD=arduino_uno_click;mikroe_can_fd_6_click
```

Comment:
Why do you need to bring in Arduino uno click in the mix?

Reply:
Yes, I got that feeling too. This additional "Shield Stack Test" is not well placed as "build_all" for drivers. Can anyone here suggest a good place for it? Perhaps an existing test suite?

If there isn't one yet, I would be very grateful for any suggestions on where such tests should be created under "tests/".